submissions/chernetskyi8704/Tiny JS World/index.js

```diff
@@ -0,0 +1,63 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+class Inhabitant {
+  constructor(species, name, gender, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+  }
+
+  displayData() {
+    return ["species", "name", "gender", "saying"]
+      .map((data) => this[data])
+      .join("; ");
+  }
+}
+
+class Person extends Inhabitant {
+  constructor(species, name, gender, saying, hands, legs) {
+    super(species, name, gender, saying);
+    this.hands = hands;
+    this.legs = legs;
+  }
+
+  displayData() {
+    return `${super.displayData()}; ${this.hands}; ${this.legs};`;
```

Comment:
Please make this (and other similar fragments in subclasses) DRYer with `Array#join`.
Do not use strings representing propety names. KISS

Reply:
Thanks a lot. Now really code looks better.